docs/CHANGELOG.asciidoc

```diff
@@ -38,6 +38,33 @@ Review important information about the {kib} 8.4.x releases.
 
 Review the following information about the {kib} 8.4.0 release.
 
+[float]
+[[known-issue-8.4.0]]
+=== Known issues
+
+Do not upgrade Kibana from 8.3 to 8.4 if you have any alerting rules with a snooze notification configured.
+
+You can identify snoozed rules in {kib} by navigating to **Stack Management** -> **Rules and Connectors**.
+Filter the rule list by selecting **View** -> **Snoozed**.
+Cancel the snooze for all affected rules before upgrading. This must be done for each Space.
+
+You can also identify snoozed rules in any Space in {kib} **Dev Tools**.
+Run the following query and searching for the string `is_snoozed_until`.
+
+[source,console]
+----
+GET kbn:api/alerting/rules/_find
+----
```

Comment:
Alternatively, we could run an ES query via the console to only get snoozed rules:

```
GET .kibana/_search
{
  "query": {
    "exists": {
      "field": "alert.isSnoozedUntil"
    }
  }
}
```

Reply:
Another reason to use the ES query is the kbn command will only list rules in the default space. Users would need to run GET kbn:s/<spaceId>/api/alerting/rules/_find to find rules in each of their spaces

Reply:
Does this return results from all spaces?

Reply:
Perfect!

Reply:
Yes, the ES query will work across spaces

Reply:
Updated!